Create ui-c8y-1021-25-3-done-steps-in-stepper-are-not-properly-marked-when-user-navigates.md (#2722)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-25-3-done-steps-in-stepper-are-not-properly-marked-when-user-navigates.md

---
date: ""

> title: Stepper navigation marks steps incorrectly when navigating back and forth

Reviewer: ```suggestion
title: Stepper navigation marks steps correctly when navigating back and forth
```

> title: Done steps in stepper are not properly marked when user navigates back and forward (#7762) [GRAFT][release/cd] (#7787)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Stepper navigation marks steps incorrectly when navigating back and forth
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60759
version: 1021.25.3
---

> Done steps in stepper are not properly marked when user navigates back and forward (#7762) [GRAFT][release/cd] (#7787)

Reviewer 2: ```suggestion
In some scenarios, users may navigate back and forth between steps in a stepper component. Previously, when navigating this way, the stepper incorrectly marked and displayed completed steps as incomplete. This has now been fixed so that the stepper component correctly reflects the done steps when the user navigates backwards and forwards through the steps.
```

> Done steps in stepper are not properly marked when user navigates back and forward (#7762) [GRAFT][release/cd] (#7787)

Author: The Claude AI suggested to use the following body:
```suggestion
In some scenarios, users may navigate back and forth between steps in a stepper component. Previously, when navigating this way, the stepper incorrectly marked and displayed which steps had already been completed. This has now been fixed so that the stepper component correctly reflects the done steps when the user navigates backwards and forwards through the steps. This change improves the user experience and avoids confusion about which steps have been finished when using steppers with navigation.
```

> In some scenarios, users may navigate back and forth between steps in a stepper component. Previously, when navigating this way, the stepper incorrectly marked and displayed completed steps as incomplete. This has now been fixed so that the stepper component correctly reflects the done steps when the user navigates backwards and forwards through the steps.

Reviewer: ```suggestion
In some scenarios, users may navigate back and forth between steps in a stepper component. Previously, when navigating this way, the stepper incorrectly marked and displayed completed steps as incomplete. This has now been fixed so that the stepper component correctly reflects the completed steps when the user navigates backwards and forwards through the steps.
```
